AMD-MAINTENANCE-GUIDE.md

```diff
@@ -0,0 +1,338 @@
+# Awesome-MakerDAO: Maintenance Guide
+
+Awesome-MakerDAO is a community-curated list of MakerDAO-related content and resources to which anyone can contribute. The long-term goal is developing a self-maintaining document, where anyone can get paid to add or update content and find and replace typos, with clear guidelines around how to update its resources.
+
+This guide details the suggested logic contributors should follow to improve Awesome-MakerDAO.
+
+- Follow the provided guidelines to help preserve a consistent and clean formatting style.
+- Keep Awesome-MakerDAO relevant. 
+  - Add, delete, and relocate resources when appropriate.
+  - Request admin approval before removing resources or when restructuring sections.
+
+**Bounties and Incentives**
+
+The MakerDAO Community Grants program supports members that provide useful contributions and improve Maker's document base. Inquire about bounties and incentives in the #Community-Development channel on MakerDAO's [Rocket.Chat](https://chat.makerdao.com).
+
+**AMD Documents:**
+
+- [Live](https://awesome.makerdao.com/)
+
+## AMD Maintenance Standards
```

Comment:
Can probably remove the AMD and just call this 'Maintenance Standards'

Reply:
That would flow better... but since I anticipate there'll be lots of maint guides for random docs... each will have their own standards? Is a distinction worth making? I can't imagine any confusion either way though

Reply:
made this change